README.md

```diff
@@ -9,7 +9,6 @@
 
 👤 **karthick harimoorthy**
 
```

Comment:
# Improve on the README
 - [ ] The project's README has to be as descriptive as possible ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)). KIndly add instruction on here on how to well clone and run the code.